Sources/XCTest/XCTestObservation.swift

```diff
@@ -77,3 +77,13 @@ public extension XCTestObservation {
     func testSuiteDidFinish(_ testSuite: XCTestSuite) {}
     func testBundleDidFinish(_ testBundle: NSBundle) {}
 }
+
+/// Expanded version of `XCTestObservation` used internally to respond to
+/// additional events not publicly exposed.
+internal protocol _XCTestObservation: XCTestObservation {
```

Comment:
Yeah, that probably makes sense, in particular considering that we'd like to make the public/internal separation of files more explicit in the file system structure. Thoughts on the name of the file? I feel like having a prefixed underscore on a source filename wouldn't be so great.